.gitignore

```diff
@@ -1,5 +1,2 @@
 node_modules
 _agstate
-# exclude everything in the generated directory except the README.md
-ui/src/generated/**
```

Comment:
For some reason these defaults were missing and causing CI to fail https://github.com/Agoric/dapp-treasury/runs/4603181466

I'm not sure why it worked before, but I think checking them in once to fix it is okay. We already do this in card-store for example https://github.com/Agoric/dapp-card-store/blob/main/ui/src/conf/defaults.js. The alternative is to change the workflows to run `ui/use-on-chain-config.js` which just write the same thing to the file. However, users can already override this file with their own deploy scripts so I don't think it matters.